CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+
+## Unreleased
+
+### Improvements/Changes
+
+### Accessibility
+
+## Bug Fixes Users Might Notice
+
+## Behind the Scenes Changes Users Probably Won't Notice
+- Add CI check to include changelog message in change requests (#5836)
+
+## RC 173 - 2022-01-13
+
+### Improvements/Changes
+- Authentication: Limit maximum number of phone numbers (LG-5493) (#5779)
+
+### Behind the scenes bug fixes users probably won't notice
```

Comment:
supernit, some ideas about changing the headings so that it's easy to tell the various sections apart?

```suggestion
Changelog
========

Unreleased
-----------

### Improvements/Changes

### Accessibility

### Bug Fixes Users Might Notice

### Behind the Scenes Changes Users Probably Won't Notice
- Add CI check to include changelog message in change requests (#5836)

RC 173 - 2022-01-13
----------------------

### Improvements/Changes
- Authentication: Limit maximum number of phone numbers (LG-5493) (#5779)

### Behind the scenes bug fixes users probably won't notice
```

Reply:
and to be clear, it's just a syntax change since `===` is h1 and `---` is h2 (and also it updated some of the h3s to be h3s, but that's neither here nor there)

Reply:
I also snuck in a section change from `### Behind the Scenes Bug Fixes Users Probably Won't Notice` to change `Bug Fixes` to `Changes` since most that end up there aren't necessarily bug fixes.

I'll have some handbook updates too for these changes.